xml/System.Windows.Forms/TableLayoutSettings.xml

```diff
@@ -357,7 +357,8 @@
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentException">The property value is not valid for the enumeration type.</exception>
+        <exception cref="T:System.ArgumentException">.NET 5 and earlier versions only: The property value is not valid for the enumeration type.</exception>
```

Comment:
The [breaking change issue](https://github.com/dotnet/docs/issues/21239) said it was ArgumentOutOfRangeException. Should this be changed?

Reply:
It was `ArgumentException` in .NET Framework, but was changed in .NET Core 3.0 timeframe (https://github.com/dotnet/winforms/pull/665), which was before my time and thus went undocumented. And now we're changing it again :)